src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -21,142 +21,153 @@ signupBanner:
 
-It may take several minutes after enabling the browser monitoring agent before your webpage data appear in New Relic. If have problems, follow our [troubleshooting tips](/docs/browser/browser-monitoring/troubleshooting/troubleshoot-your-browser-monitoring-installation/).
+  <InlineSignup/>
 
-## Deployment options [#options]
+    If you already have an account, sign in and move onto the next step. 
 
-No matter which option you use to deploy browser monitoring, the end result is the same: the browser monitoring JavaScript snippet (also referred to as the "agent") is inserted into your app pages. The method you select depends on your preferences and business needs.
+<Steps>
 
-To learn more about the scripts used to install the agent, check out the browser agent's [Read me](https://github.com/newrelic/newrelic-browser-agent) file in our GitHub repo. You can also learn about the third party libraries called during installation in the [Third party notices](https://github.com/newrelic/newrelic-browser-agent/blob/main/THIRD_PARTY_NOTICES.md) file. 
+<Step>
+### Add a browser app in the New Relic UI
 
-### Enable automatically on APM-monitored app [#select-apm-app]
+First, you'll need to add your browser app to the New Relic UI:
 
-When [enabling browser monitoring](#enable), you can use an <InlinePopover type="apm" /> agent to automatically inject the browser monitoring JavaScript snippet for you. This is the easiest way to install the agent for an app that's already being monitored by APM.
+Go to **[one.newrelic.com](https://one.newrelic.com)**, click **Add data**. Under **Data sources** select **Browser monitoring**. 
 
-APM-monitored apps are listed on your [APM **Applications** index](/docs/apm/apm-ui-pages/monitoring/view-your-applications-index/).
+<img
+  title="Add your app"
+  alt="Add your browser app in the UI"
+  src={browserInstallStep1}
+  style={{ maxWidth: '70%' }}
+/>
 
-### Enable with copy/paste [#copy-paste-app]
+<Callout variant="tip">
+Need to add your browser app via an API? You can [use our Nerdgraph API to create a browser app](/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph#create-browser).
+  </Callout>
+</Step>
+<Step>
+### Choose a deployment method [#step-3]
 
-When [enabling browser monitoring](#enable), you can manually insert the JavaScript snippet into your app's webpages. The copy/paste option gives you control over the exact placement of our JavaScript snippet, which is required to monitor the webpage's performance. This is useful for:
+When it comes to installing the browser agent, you've got two options:
 
-* Standalone apps, static sites, and cached pages delivered by CDN
-* <InlinePopover type="apm" /> apps that are not as closely coupled to the browser app as with a standard server-side app (for example, when your client-side app talks to a REST API back end)
+* Option 1: Auto-instrument via a New Relic APM agent (recommended for non-SPA apps)
```

Comment:
How about "Deployment method 1" and "Deployment method 2"?

Reply:
Actually, why not just map to the text in the screenshots:

"Enable via New Relic APM"
"Copy/Paste JavaScript code"